build.sbt

```diff
@@ -25,14 +25,16 @@ lazy val sharedSettings = MimaSettings.settings ++ scalaVersionSettings ++ Seq(
 
   name := "scalacheck",
 
+  isRelease := false,
```

Comment:
I'm assuming that the command-line overrides this?

Reply:
Yes, exactly.